### PR TITLE
action external_output given custom output in statement

### PR DESCRIPTION
**Pull request by seabassdk**
http action changed a statement to: "I connect to {} and save the output into {}"
Implemented changes to handle key with output name.

### CODE REVIEW

**Comment by puria** [comment]
@seabassdk 🤟

File: packages/http/src/index.js [hunk 1]
```diff
@@ -33,17 +33,19 @@ export default (req, res, next) => {
     }
 
     if (zencode.match(ACTIONS.EXTERNAL_OUTPUT)) {
+      const outputNames = zencode.paramsOf(ACTIONS.EXTERNAL_OUTPUT);
       if (!externalSourceKeys.length)
         throw new Error(`[HTTP]
-              Endpoints are missing, please define them with the 
-              following zencode sentence "${ACTIONS.EXTERNAL_CONNECTION}"`);
+            Endpoints are missing, please define them with the
```
Comment (seabassdk) [inline packages/http/src/index.js:39]: I dont know why this changed....

**Comment by puria** [comment]
Would you mind to update the test also?

**Review by puria — APPROVED** [review]

**Comment by seabassdk** [comment]
Yeah no problem i'll update the tests

**Comment by puria** [comment]
Just tried with the following contract (see https://github.com/dyne/restroom-mw/pull/19/commits/6117344fe62e427d059c9c7e03d41c8cd06d5649): 

```gherkin
Rule unknown ignore
Scenario 'ecdh': Create the keypair
Given that I have an endpoint named 'endpoint'
Given I connect to 'endpoint' and save the output into 'output'
Given I have a 'string dictionary' named 'output' 
Then print the 'output'
```

and the following data

```json
{ "endpoint": "https://apiroom.net/api/dyneorg/Create-a-keypair" }
```

And does not work, am I missing something?

**Comment by puria** [comment]
superseeded by #25

**Pull request closed without merging**